web/src/components/Layout/NewAppDialog/index.tsx

```diff
@@ -99,7 +105,7 @@ export const NewAppDialog = memo(function NewAppDialog(
               className="w-full font-rubik"
               placeholder="For internal reference. Visible only to you and your team."
               type="text"
-              {...register("description_internal")}
```

Comment:
the placeholder above is wrong, there may also be other indications in the UI that are wrong